packages/liveblocks-client/src/types/index.ts

```diff
@@ -264,6 +264,7 @@ export type ClientOptions = {
   throttle?: number;
   fetchPolyfill?: any;
   WebSocketPolyfill?: any;
+  atobPolyfill?: (data: string) => string;
```

Comment:
Thinking out loud… would this be a good time to restructure the options a bit, into this shape:

```ts
export type Polyfills = {
  fetch?: any;
  WebSocket?: any;
  atob?: (data: string) => string;
};

export type ClientOptions = {
  throttle?: number;
  polyfills?: Polyfills;
} & (
  | { publicApiKey: string; authEndpoint?: never }
  | { publicApiKey?: never; authEndpoint: AuthEndpoint }
);
```

I think that might make it a bit more readable and obvious what these are for. If we do so, we should definitely keep the existing two (`fetchPolyfill` and `WebSocketPolyfill`) on the ClientOptions in 0.17, but we could deprecate them and remove them in 0.18.

Reply:
This won't have to be part of this PR, btw! We could do that in a separate one.

Reply:
Since we'll want to include this change with the 0.17 release, I think it'll be nice to also include this suggested reorganization into a `polyfills` sub-object. I have a item on my 0.17 checklist for that (see #303), so we won't forget about it.